boxEngine.py

```diff
@@ -0,0 +1,182 @@
+
+        def NotImplement():
+            warn("Sorry, but this function is not implemented yet!")
+
+        def ConfigFatal(Err):
+            console.print_exception(f"[bold][bright_red][Error!] Config file or an element is missing! -> {Err}")
+        
+        def data404():
+            error("Data needed to access wasn't found.")
+
+        def userFail():
+            error("Input Password failed too many times.")
+
+        def user404():
+            error("User not found.")
+
+    class funct:
+        def echo(x):
+            print(x)
+        
+        def shellVer():          
+            print(f"boxpyshell created by ValkTheBoxman\nver: {ver} Build: main\n（＾ω＾）\n")
+
+
+    class pc:
+        def info():
+            # get pc info
+            uname = platform.uname()
+            print(f"CPU bit:{platform.machine()}\n",f"PC OS: {platform.platform}\n",f"PC Nodename:{uname.node()}")
+
+class animlib:
+    def loadingAnim(x,y) -> None:
+
+        top = "|"
+        right = ">"
+        left = "<"
+
+        if x == "load":
+            state = "loading...  "
+        elif x == "exit":
+            state = "exiting... "
+
+        for i in range(y):
+            print(state,top, end="\r")
+            time.sleep(0.5)
+            print(state,left, end="\r")
+            time.sleep(0.5)
+            print(state,top, end="\r")
+            time.sleep(0.5)
+            print(state,right, end="\r")
+            time.sleep(0.5)
+
+
+
+class pythonBasic:
+    class functions:
+        def length():
+            print("Count the letters in a word!")
+            string = input()
+            len(string)
+        def identify():
+            string = input()
+            id(string)
+            
+
+    class math:
+        def add():
+            num1 = input("What is your first number? -> ")
+            num2 = input("What is your second number? -> ")
+            print(f"adding {num1} + {num2}")
+            print(int(num1) + int(num2))
+        
+        def subtract():
+            num1 = input("What is your first number? -> ")
+            num2 = input("What is your second number? -> ")
+            print(f"subtracting {num1} - {num2}")
+            print(int(num1) - int(num2))
+
+    class fun:
+        def helloUser():
+            print("What is your name?")
+            name = input()
+            print(f"Hello!, {name}")
+        
+        def eightBall():
+            print("Ask me a question and i shall give you an answer")
+            ques = input(" -> ")
+            list = ["Outlook Good","Outlook not so good","i cannot answer at this moment","It is certain.","It is decidedly so.","Without a doubt.","Yes definitely.","You may rely on it.","As I see it, yes.Most likely.","Yes.","Signs point to yes.","Reply hazy, try again.","Ask again later.","Better not tell you now.","Cannot predict now.","Concentrate and ask again.","Don't count on it.","My reply is no.","My sources say no.","Very doubtful."]
```

Comment:
I would personally put it in a file but this is a personal thing

Reply:
not going to do it